embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt

```diff
@@ -48,7 +48,7 @@ internal class V2PayloadStore(
             System.NativeCrash.value -> SupportedEnvelopeType.CRASH
             System.ReactNativeCrash.value -> SupportedEnvelopeType.CRASH
             System.FlutterException.value -> SupportedEnvelopeType.CRASH
-            System.NetworkCapturedRequest.value -> SupportedEnvelopeType.NETWORK
+            System.NetworkCapturedRequest.value -> SupportedEnvelopeType.BLOB
```

Comment:
IMO this terminology is somewhat overloaded with AEI

Reply:
Yeah fair. These all contain blobs but AEI has a blob within a blob, and soon, crashes will too.

I'll ask the group to see what folks like and we can change it after